src/lib/parameters/parameters.cpp

```diff
@@ -651,10 +652,10 @@ param_autosave()
 	// - tasks often call param_set() for multiple params, so this avoids unnecessary save calls
 	// - the logger stores changed params. He gets notified on a param change via uORB and then
 	//   looks at all unsaved params.
-	hrt_abstime delay = 300 * 1000;
+	hrt_abstime delay = 300_ms;
 
-	const hrt_abstime rate_limit = 2000 * 1000; // rate-limit saving to 2 seconds
-	hrt_abstime last_save_elapsed = hrt_elapsed_time(&last_autosave_timestamp);
+	static constexpr const hrt_abstime rate_limit = 2_s; // rate-limit saving to 2 seconds
```

Comment:
`static constexpr const` is a bit too much here, no? Does it actually make a difference?

Reply:
Ha, in this case it almost certainly not, but I guess the point would be that it's never going to be worse to make something a compile time constant.

Reply:
True, I was just thinking that it hurts readability a bit.